docs/rules/prefer-effect-callback-in-block-statement.md

```diff
@@ -0,0 +1,38 @@
+# Prefer Effect Callback In BlockStatement
+
+> A block statement is easier to troubleshoot
+
+## Rule Details
+
+This rule prefers that the callback of an effect is a block statements.
+This makes it easier to troubleshoot type errors, for when example an RxJS operator isn't imported.
+
+Examples of **incorrect** code for this rule:
+
+```ts
+class Effect {
+  effectNOK = createEffect(() =>
```

Comment:
Maybe change to `effectNotOk`? I was wondering for a minute if maybe NOK is some NgRx terminology  unknown to me 😄

Reply:
I'm going to merge this in, I believe other are also using NOK.
I'll refactor that later and all in one PR 🙂
Thanks for the review!